### PR TITLE
feat: add support to add specific scanner to registry create/update/delete

### LINKED ISSUE #152: Creating a new Registry Integration in Aqua Cloud doesn't support specific scanners
With the 0.8.12 release, we can now add a registry integration in Aqua Cloud, thanks to the addition of scanner_type in the provider
e.g. https://github.com/aquasecurity/terraform-provider-aquasec/blob/main/client/registries.go#L41

This is equivalent to configuring the registry integration as in this screenshot:
image

Yet, some clients want to use specific scanners for the registry, and this configuration is still not available:
image

### LINKED ISSUE #164: Terraform is always trying to change the "aquasec_user/first_time" parameter when user logs in for the first time
I'm provisioning the Aqua user using this code to force users to change their password when they login for the first time:

```hcl
resource "aquasec_user" "test_user" {
  user_id    = "test-user"
  password   = "Test123,."
  roles = [
    "Auditor"
  ]
  email      = "test@test.com"
  first_time = true              ### <<<--- This is the parameter
  name       = "test-user"
}
```

It is working fine and users must change their password when they login for the first time.
Once they do it and I run the code again then the Terraform wants to change back the `first_time = true` - so I'm loosing the idempotency of the code:

```
...
aquasec_user.test_user: Refreshing state... [id=test-user]

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # aquasec_user.test_user will be updated in-place
  ~ resource "aquasec_user" "test_user" {
      ~ first_time = false -> true
        id         = "test-user"
        name       = "test-user"
        # (8 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

### Terraform Version

```bash
$ terraform -v
Terraform v1.2.5
on darwin_amd64
+ provider registry.terraform.io/aquasecurity/aquasec v0.8.12
```

### Affected Resource(s)
- aquasec_user

### Expected Behavior
The code+provider should keep the idempotency - which means no matter how many times you run the terraform code it must give you the same results without reporting any changes... 

### Actual Behavior
When running the code above multiple times (after users logged in for the first time) I can see TF wants to do the change each run.

### Steps to Reproduce
Code is above

### PR DESCRIPTION
**Pull request by KoppulaRajender**
fix #152 
fix #164 

---
update docs

### CODE REVIEW

File: aquasec/resource_registry.go [hunk 1]
```diff
@@ -83,10 +83,43 @@ func resourceRegistry() *schema.Resource {
...
+			"scanner_name": {
+				Type:        schema.TypeList,
+				Description: "List of scanner names",
+				Optional:    true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"scanner_name_added": {
+				Type:        schema.TypeList,
```
Comment (yossig-aquasec) [inline aquasec/resource_registry.go:95]: Are the added, removed are only internal or user can set them manual, if its internal you can removed them from the scheme

Comment (KoppulaRajender) [reply aquasec/resource_registry.go:95]: only internal, users input is not required. I tried adding them as 'computed' but failing test cases, as the terraform plan was not empty and expecting some values. In real the creation api does not have this variables.

Comment (yossig-aquasec) [reply aquasec/resource_registry.go:95]: you don't need to add them to the scheme at all

Comment (KoppulaRajender) [reply aquasec/resource_registry.go]: thanks, removed from the schema

**Review by yossig-aquasec — APPROVED** [review]
